docs/net-comparisons.md

```diff
@@ -0,0 +1,35 @@
+# How is lokinet different than ...
+
+
+## Tor Browser
+
+Tor browser is a hardened Firefox Web Browser meant exclusively to surf http(s) sites via Tor. It is meant to be a complete self contained browser you open and run to surf the Web (not the internet) anonymously.
+Lokinet does not provide a web browser at this time because that is not a small task to do at all, and an even larger task to do in a way that is secure, robust and private. Community Contribuitions Welcomed.
+
+## Tor/Onion Services
+
+While Tor Browser is the main user facing product made by Tor Project the main powerhouse is Tor itself which provides a way to anonymize tcp connections made by an inititor and optionally additionally the recipiant in the case of when using a .onion address. Lokinet provides a similar feature set but can carry anything that can be encapsulated in an IP packet (currently only unicast traffic). 
```

Comment:
inititor -> initiator
recipiant -> recipient

"in the case of when using" -> "when using"